### PR TITLE
Removal of SortedOracle multiplier

### PR DESCRIPTION
**Pull request by pahor167**
### Description

Removal of multiplier in SortedOracles since we will not be using it. 

Addresses: https://github.com/celo-org/audit-a-findings/issues/6
Addresses: https://github.com/celo-org/audit-a-findings/issues/15

### CODE REVIEW

File: packages/protocol/contracts/stability/SortedOracles.sol [hunk 1]
```diff
@@ -351,16 +344,10 @@ contract SortedOracles is ISortedOracles, ICeloVersionedContract, Ownable, Initi
   function medianRate(address token) external view returns (uint256, uint256) {
     EquivalentToken storage equivalentToken = equivalentTokens[token];
     if (equivalentToken.token != address(0)) {
-      (uint256 equivalentMedianRate, uint256 numRates) = medianRateWithoutEquivalentMapping(
+      (uint256 equivalentMedianRate, uint256 _numRates) = medianRateWithoutEquivalentMapping(
```
Comment (m-chrzan) [inline packages/protocol/contracts/stability/SortedOracles.sol]: `_numRates` => `denominator`. And looks like this is incorrectly named in several other places (e.g. the `medianRate` function).

**Review by m-chrzan — COMMENTED** [review]
There's still a reference to `multiplier` in the comment for `getEquivalentToken`.

File: packages/protocol/contracts/stability/SortedOracles.sol (hunk 1 above)
Comment (pahor167) [reply packages/protocol/contracts/stability/SortedOracles.sol]: Fixed, thanks

**Comment by pahor167** [comment]
> There's still a reference to `multiplier` in the comment for `getEquivalentToken`.

Thanks, fixed

**Review by soloseng — APPROVED** [review]

**Pull request merged**